assets/sass/_theme/sections/locations.sass

```
@@ -1,51 +1,76 @@
 .locations
-    &--list
-        .location + .location
+    .location
+        display: flex
+        flex-direction: column-reverse
+        gap: calc(var(--grid-gutter) / 2) var(--grid-gutter)
```

```suggestion
gap: calc(var(--grid-gutter) / 2)
```
(valable pour mobile + grid)